packages/typegpu/src/core/texture/externalTexture.ts

```diff
@@ -38,4 +38,8 @@ export class TgpuExternalTextureImpl implements TgpuExternalTexture {
 
     return id;
   }
+
+  toString() {
+    return `${this.resourceType}: ${this.label ?? '<unnamed>'}`;
```

Comment:
Removed the space to be more consistent with the rest.
```suggestion
    return `${this.resourceType}:${this.label ?? '<unnamed>'}`;
```